Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#
+FROM python:2
+
+MAINTAINER "Apache CloudStack" <dev@cloudstack.apache.org>
+LABEL Description="Apache CloudStack CloudMonkey; Python based CloudStack command line interface"
+LABEL Vendor="Apache.org"
+LABEL License=ApacheV2
+LABEL Version=5.3.2
+
+#COPY dist/cloudmonkey-*.tar.gz /
+COPY . /cloudstack-cloudmonkey
+#WORKDIR /cloudmonkey
+#RUN pip install /cloudmonkey-*.tar.gz
+#RUN rm /*.tar.gz
+RUN pip install requests
+RUN (cd /cloudstack-cloudmonkey; python setup.py build)
+RUN (cd /cloudstack-cloudmonkey; python setup.py install)
```

Comment:
I was originally using package but it seams to be against the latest tag philosophy and automated build. In automated build, it download source from github anyway so I thought it was a better method and also it can be build on commit and from source it will contain commit changes. 

Because '''FROM python:2''' the images is ~750MB